Automattic/jetpack
# Stats: Make Odyssey Stats toggle visible for all

#### Changes proposed in this Pull Request:


Makes the toggle for enabling Odyssey Stats visible for all users.

#### Other information:


#### Jetpack product discussion


Part of pejTkB-5e-p2.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:

* Apply this change to your Jetpack site connected to WordPress.com.
* Navigate to the Traffic tab of the Jetpack Settings page.
* Click on the toggle to expand the folded card:
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/4044428/209877543-bc4b217f-df59-4916-b671-cacb45249eef.png">

* Ensure that the new toggle is visible and functional.
<img width="1061" alt="image" src="https://user-images.githubusercontent.com/4044428/209877800-9042a872-5ec5-4792-a24e-a982f1b3aa7c.png">

* Enable the toggle and open the Jetpack Stats page (`/wp-admin/admin.php?page=stats`).
* Ensure that the page renders as expected.
* Ensure that the page functions as expected (e.g., no broken links or render errors).
* Please report broken functionality in [the project board](https://github.com/orgs/Automattic/projects/484).
